Doc/library/codecs.rst

```diff
@@ -1024,9 +1024,9 @@ particular, the following variants typically exist:
 | cp500           | EBCDIC-CP-BE, EBCDIC-CP-CH,    | Western Europe                 |
 |                 | IBM500                         |                                |
 +-----------------+--------------------------------+--------------------------------+
-| cp720           |                                | Arabic                         |
+| cp720           | 720                            | Arabic                         |
```

Comment:
Can you please add ".. versionchanged:: 3.7" markup after the table to mention that you added aliases to cp720, cp737, ... ?

Reply:
@vstinner You mean "".. versionchanged:: 3.8", no?

Reply:
Oh... right :-) Sorry our 3.7 release manager!

Reply:
Sorry about that. Fixed with 27d9380c8738de7673de316121711665f3e8ac49 . Thanks.